packages/destination-actions/src/destinations/google-enhanced-conversions/functions.ts

```diff
@@ -69,3 +83,18 @@ export function convertTimestamp(timestamp: string | undefined): string | undefi
   }
   return timestamp.replace(/T/, ' ').replace(/\..+/, '+00:00')
 }
+
+export function get_api_version(features: Features | undefined, statsContext: StatsContext | undefined): string {
```

Comment:
We use camel casing for function names. Also features? is slightly more readable than features | undefined
```suggestion
export function getAPIVersion(features?: Features, statsContext?: StatsContext): string {
```